spec/services/open_library_service_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'OpenlibraryService', type: :service do
```

Comment:
I'm more interested in the Controller's tests

Reply:
Although these are fine too.

Reply:
These tests are focused on the open library service response, for example, this:

[ https://openlibrary.org/api/books?bibkeys=ISBN:0385472579&format=json&jscmd=data](url)